qstack/spahm/rho/bond.py

```diff
@@ -116,10 +113,13 @@ def get_repr(mols, xyzlist, guess,  xc=defaults.xc, spin=None, readdm=None,
         all_atoms   = np.array([z for mol in mols for z in mol.elements if z in only_z], ndmin=2)
     else:
         all_atoms   = np.array([mol.elements for mol in mols])
+    spin = np.array(spin) ## a bit dirty but couldn't find a better way to ensure Iterable type!
```

Comment:
a pity that it has to be done twice when calling the `main()` script, but couldn't find any work around !